service-discovery/README.adoc

```diff
@@ -726,9 +776,190 @@ When you create the binding connection, the Service Binding Operator injects con
+* The RHOAS Operator is installed on your OpenShift cluster and you've verified connection to the cluster. See link:{base-url}{service-binding-url}#proc-verifying-connection-to-openshift-cluster_{context}[Verifying connection to your OpenShift cluster].
+* You've connected a Kafka and {registry} instance to a project in your OpenShift cluster. See link:{base-url}{service-binding-url}#proc-connecting-kafka-registry-instance-to-openshift-cluster_{context}[Connecting a Kafka or {registry} instance to your OpenShift cluster].
+
+[id="proc-kafka-registry-deploying-example-quarkus-application-on-openshift_{context}"]
+=== Deploying an example Quarkus application on OpenShift
+
+[role="_abstract"]
+In this step of the tutorial, you deploy an example Quarkus application in the OpenShift project that you previously connected your Kafka and {registry} instances to.
+
+The Quarkus application generates random movie names and produces those names to a Kafka topic. Another part of the application consumes the names from the Kafka topic. Finally, the application uses __server-sent events__ to expose the numbers as a REST UI. A web page in the application displays the exposed names.
+
+The example Quarkus application uses link:https://quarkus.io/guides/deploying-to-kubernetes#service-binding[service binding extensions^] for Kafka, {registry}, and the Service Binding Operator. These extensions enable the application to automatically detect and use the injected connection parameters, eliminating the need for manual configuration of the application.
+
+.Prerequisites
+* You have privileges to deploy applications in the OpenShift project that you connected your Kafka and {registry} instances to.
+
+.Procedure
+
+. If you're not already logged in to the OpenShift CLI, log in using a token, as described in link:{base-url}{service-binding-url}#proc-verifying-connection-to-openshift-cluster_{context}[Verifying connection to your OpenShift cluster]. Log in as the same user who verified connection to the cluster.
+
+. Use the OpenShift CLI to ensure that the current OpenShift project is the one that you previously connected your Kafka and {registry} instances to, as shown in the following example.
++
+.Using the OpenShift CLI to specify the current OpenShift project
+[source]
+----
+$ oc project my-project
+----
+
+. To create the Quarkus application, deploy a container image provided by {org-name}.
++
+.Deploying an example Quarkus application
+[source,options="nowrap"]
+----
+$ oc new-app quay.io/secondsun/kafka-avro-schema-quickstart
+
+imagestream.image.openshift.io "kafka-avro-schema-quickstart" created
+deploymentconfig.apps.openshift.io "kafka-avro-schema-quickstart" created
+service "kafka-avro-schema-quickstart" created
+route.route.openshift.io "kafka-avro-schema-quickstart" created
```

Comment:
See https://github.com/redhat-developer/app-services-guides/pull/383/files#r778755633. These names will be reviewed and updated as necessary when we deliver the finalized app.